post_build.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
```

Comment:
Can you explain how this shell script relates to the NimBLE change?

Reply:
This is probably made after you finish compiling hence the "post_build.sh" will be deleting that.